File: evals/fixtures/java_spring_impact/src/main/java/com/example/controller/CheckoutController.java
package com.example.controller;

import com.example.service.PricingService;

public class CheckoutController {
    private final PricingService pricing;

    public CheckoutController() {
        this.pricing = new PricingService();
    }

    public double quoteTotal(double subtotal, String tier) {
        return subtotal * pricing.rateFor(tier);
    }
}

File: evals/fixtures/php_laravel_impact/app/Http/Controllers/CheckoutController.php
<?php

namespace App\Http\Controllers;

use App\Services\PricingService;

class CheckoutController
{
    public function quoteTotal(float $subtotal, string $tier): float
    {
        $service = new PricingService();
        return $subtotal * $service->rateFor($tier);
    }
}
